.github/workflows/pr_scan.yml

```diff
@@ -36,24 +36,24 @@ jobs:
       run: |
         Scripts/validate-Adyen-SDK-version.sh
 
-    - name: Clean Build and Unit Test
+    - name: Clean Build and Test
       run: |
         xcodebuild -version
         xcodebuild clean build test -project "${project}" -scheme "${scheme}" ${params} -destination "${destination}" | xcpretty --utf --color && exit ${PIPESTATUS[0]}
       env:
         project: 'Adyen.xcodeproj'
         params: '-derivedDataPath ./DerivedData -enableCodeCoverage YES -skipPackagePluginValidation'
-        scheme: 'AdyenUIKitTests'
+        scheme: 'AdyenUIHost'
         destination: 'name=iPhone 14,OS=16.4'
```

Comment:
you can try just `name=iPhone 14` without "OS" - this will trigger any available simulator, so we don't have to track what ever Xcode version is currently running

Reply:
In this case we really want to have a specific iOS version to replicate our local environment to reduce potential inconsistencies as much as possible

Reply:
> `xcode-version: '14.3'`

Ok, make sense